org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/queues/beans/TaskBean.java

```diff
@@ -21,57 +21,57 @@
 import org.eclipse.scanning.api.event.queues.models.arguments.QueueValue;
 
 /**
- * TaskBean is a type of {@link QueueBean} implementing an 
- * {@link IHasAtomQueue}. As a {@link QueueBean} it can only be passed 
+ * TaskBean is a type of {@link QueueBean} implementing an
+ * {@link IHasAtomQueue}. As a {@link QueueBean} it can only be passed
  * into the job-queue of an {@link IQueueService} and as such provides the most
- * abstract description of an experiment. 
- * 
+ * abstract description of an experiment.
+ *
  * This class of bean is used to describe a complete experiment, including the
- * beamline setup, data collection and post-processing steps. It should also 
- * contain all the sample metadata necessary to write the NeXus file. 
- * 
+ * beamline setup, data collection and post-processing steps. It should also
+ * contain all the sample metadata necessary to write the NeXus file.
+ *
  * TODO Sample metadata holder.
- * 
+ *
  * @author Michael Wharmby
  *
  */
 public class TaskBean extends QueueBean implements IHasAtomQueue<SubTaskAtom> {
 
 	/**
-	 * Version ID for serialization. Should be updated when class changed. 
+	 * Version ID for serialization. Should be updated when class changed.
 	 */
 	private static final long serialVersionUID = 20161021L;
-	
+
 	private LinkedList<SubTaskAtom> atomQueue;
 	private LinkedList<QueueValue<String>> queueAtomShortNames;
 	private String queueMessage;
 //	private Object nexusMetadata; TODO!!!!
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)